pycbc/events/coinc.py

```diff
@@ -1212,7 +1209,7 @@ def _find_coincs(self, results, valid_ifos):
             offsets = numpy.concatenate(offsets)
             ctime0 = numpy.concatenate(ctimes[self.ifos[0]]).astype(numpy.float64)
             ctime1 = numpy.concatenate(ctimes[self.ifos[1]]).astype(numpy.float64)
-
+            logging.info("Clustering %s-%s coincs", self.ifos[0], self.ifos[1])
```

Comment:
Import and use the `ppdets()` function here.

Reply:
from detector.py?